### PR TITLE
Feature start creating subplan

### PR DESCRIPTION
**Pull request by joshcbee**
Now possible to navigate from home to the tool creator (entering identification details to be implemented separately).

### CODE REVIEW

File: cassdegrees/templates/createsubplan.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+{% block page-title %}Create Subplan{% endblock %}
+
+{% block subtitle %}Create Subplan{% endblock %}
+
+{% block content %}
+
+<p>This is a test</p>
+
+{% endblock %}
```
Comment (j-selby) [inline cassdegrees/templates/createsubplan.html]: Newline.

**Review by j-selby — APPROVED** [review]
Looks good!

**Review by JackSekoranja — APPROVED** [review]
Looks Good

**Review by Daniel-Jang — APPROVED** [review]

**Comment by JackSekoranja** [comment]
Added James' suggestion and fixed merge conflicts. Pull Request is good to go!

File: cassdegrees/templates/createsubplan.html (hunk 1 above)
Comment (j-selby) [reply cassdegrees/templates/createsubplan.html]: This still exists in the current version of the application (newline).

**Review by j-selby — APPROVED** [review]
Looks good.

**Pull request merged**